custom_components/magiqtouch/magiqtouch.py

```diff
@@ -46,29 +43,15 @@
 WebServiceURL = "https://57uh36mbv1.execute-api.ap-southeast-2.amazonaws.com/api/"
 ApiUrl = (
     "https://57uh36mbv1.execute-api.ap-southeast-2.amazonaws.com"
-    + "/api/loadmobiledevice"
+    + "/api/"
 )
 
-_LOGGER = logging.getLogger("magiqtouch")
-
-# from https://docs.aws.amazon.com/iot/latest/developerguide/server-authentication.html
-rootCAPath = Path(__file__).parent / "SFSRootCAG2.pem"
-
-# # MQTT logging
-# logger = logging.getLogger("AWSIoTPythonSDK.core")
-# logger.setLevel(logging.DEBUG)
-# streamHandler = logging.StreamHandler()
-# formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-# streamHandler.setFormatter(formatter)
-# logger.addHandler(streamHandler)
-
-# from aiobotocore import credentials
-
-# class AioCredentialResolver(credentials.CredentialResolver):
-#     async def load_credentials(self):
-#         return None
+NewWebApiUrl = (
+    "https://tgjgb3bcf3.execute-api.ap-southeast-2.amazonaws.com/prod"
+    + "/v1/"
+)
```

Comment:
Could you share where this url came from? If nothing else a note will help to find a new URL in future if it's charged.

Reply:
This was sniffed from the iOS app (no idea if it would be any different on Android). Probably good to leave a note there, I agree.